tests/tests/class-events-store-tests.php

```diff
@@ -148,33 +148,6 @@ function test_event_unscheduling_against_event_store() {
 		$this->assertEmpty( Utils::get_events_from_store() );
 	}
 
-	/**
-	 * Test event-cache splitting
-	 */
-	function test_excessive_event_creation() {
```

Comment:
There is no pre-cache-bucketing anymore, so we don't need to worry about this. I've made a note to remove this later: https://github.com/Automattic/vip-go-mu-plugins/blob/a361f213b517eb3e6a070576bb8bf34e1dafbb65/001-cron.php#L118-L126